content/3_Silver/Binary_Search_Ans.mdx

```diff
@@ -105,7 +105,7 @@ export const problems = {
 			'Normal',
 			true,
 			[],
-			'binary search on time with epsilon and then find if there exist non-empty intersection of the ranges (maximum of left points and minimum of right points)'
+			''
```

Comment:
I think you should leave the solution sketch for now (bad solution sketch > no solution sketch)